provisioning/baremetal_pod.go

```diff
@@ -266,6 +267,33 @@ func createInitContainerIpaDownloader(images *Images) corev1.Container {
 	return initContainer
 }
 
+func createInitContainerCoreOSIPA(info *ProvisioningInfo) corev1.Container {
+	config := &info.ProvConfig.Spec
+	initContainer := corev1.Container{
+		Name:            "configure-coreos-ipa",
```

Comment:
maybe prefix with `metal3-` for consistency and discoverability?

Reply:
+1. Makes sense.